.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,64 @@
+name: Continuous Integration
+on:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - '.gitignore'
+      - '.env.template'
+      - '.env.j2'
+  push:    
+    paths-ignore:
+      - 'README.md'
+      - '.gitignore'
+      - '.env.template'
+      - '.env.j2'
+  schedule:
+    - cron: '08 13 * * *'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run unit and integration tests
+        run: docker-compose -f docker-compose.test-ci.yml run test-ci
+        if: github.event.schedule != '08 13 * * *'
```

Comment:
I might have miss-read the step. I thought it meant to setup a schedule that only runs the security scan. I can remove the if statements so that everything runs?

Reply:
Ah that makes sense - yes it's not very specific is it!

I'm happy to just stick with this - I did start writing the below but you'll see the conclusion is that I'd probably not get too worried about changing things!

If we were running _only Snyk_ on a schedule I'd be tempted to handle that as a separate "job", which probably makes things a bit easier to read:

```yml
jobs:
  build:
    name: Build and test
    runs-on: ubuntu-latest
    if: github.event.schedule != '08 13 * * *'
    steps:
      - uses: actions/checkout@v2
      - name: Run unit and integration tests
        run: docker-compose -f docker-compose.test-ci.yml run test-ci
     ...
  scan:
    name: Scan depenendencies
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Run Snyk to check for vulnerabilities
        uses: snyk/actions/python-3.10@master
        env:
          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
        ...
```

If you _did_ go this way we'd want to think about the Slack notifications though - do we send them twice (once for each job), or do we have one that covers both? If we have one that covers both, how do we make sure it fires regardless of whether or not the first job executes (since that will be skipped if it's triggered by the schedule)

So then we'd potentially need [something like this discussion on conditional jobs](https://stackoverflow.com/questions/70959792/how-to-run-github-action-job-after-all-conditional-jobs-even-its-didnt-ran)

And then we'd need to check that the job still has the relevant context to send us a helpful message at which point, I've decided it's probably more complicated than it's worth here, and I'd actually stick with your approach - but interesting thinking it through anyway! 😄